etc/profile-m-z/quodlibet.profile

```diff
@@ -0,0 +1,67 @@
+# Firejail profile for quodlibet
+# Description: Music player and music library manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include quodlibet.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/quodlibet
+noblacklist ${HOME}/.config/quodlibet
+noblacklist ${HOME}/.quodlibet
+noblacklist ${MUSIC}
+
+include allow-bin-sh.inc
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/quodlibet
+mkdir ${HOME}/.config/quodlibet
+mkdir ${HOME}/.quodlibet
+
+
```

Comment:
```suggestion
```

Reply:
@rusty-snake: Do you suggest I remove the space between `mkdir` and `whitelist`?

Reply:
yes.

Do it now or I merge it anyway.